core/src/main/java/hudson/model/ManagementLink.java

```diff
@@ -122,6 +122,16 @@ public boolean getRequiresConfirmation() {
         return Jenkins.ADMINISTER;
     }
 
+    /**
+     * Whether to show this link on /manage
+     *
+     * @return true if and only if the link should be shown
+     */
+    public boolean isShowLink() {
+        final Permission requiredPermission = getRequiredPermission();
+        return requiredPermission == null || Jenkins.get().hasPermission(requiredPermission);
```

Comment:
> `requiredPermission == null`

I don't think this is possible anymore.

Also, this method may no longer be even needed if instead checking for Overall/Read in `getRequiredPermission`.

Reply:
correct, intellij says this is impossible